rest/adminapitest/admin_api_test.go

```diff
@@ -3776,6 +3776,52 @@ func TestSetFunctionsWhileDbOffline(t *testing.T) {
 	resp.RequireResponse(http.StatusOK, syncFunc)
 }
 
+func TestCollectionSyncFnWithBackticks(t *testing.T) {
+	if base.TestsDisableGSI() {
+		t.Skip("This test only works with Collections")
+	}
+
+	base.SetUpTestLogging(t, base.LevelInfo, base.KeyHTTP)
+
+	// Get a test bucket, and use it to create the database.
+	tb := base.GetTestBucket(t)
+	defer tb.Close()
+
+	rt := rest.NewRestTester(t, &rest.RestTesterConfig{PersistentConfig: true})
+	defer rt.Close()
+
+	scopesConfig := rest.GetCollectionsConfig(t, tb, 1)
```

Comment:
`collection := rt.GetSingleTestCollection()` and then you'll have scope and and collection and name

Reply:
`GetSingleDatabaseCollection` doesn't work with persistent config before making a database, panics at https://github.com/couchbase/sync_gateway/blob/6c507c239a7903190ebf90f38ed7353dfc018c6f/db/util_testing.go#L594